README.md

```
@@ -1 +1,60 @@
+```bash
+pnpm install
+pnpm dev      # local dev server
+pnpm build    # static build → dist/
+pnpm preview  # serve the build
+```
+
+## Posting
+
+Drop a Markdown file in the right section folder and push:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)